Sources/Views/Collections/CollectionSection.swift

```diff
@@ -42,7 +42,7 @@ public func collectionSection(
       length: section.length,
       blurb: section.blurb
     ),
-    coreLessons(section.coreLessons),
+    coreLessons(collection: collection, section: section),
```

Comment:
there's a few spots we need to thread collection and section info deeper into the UI so that we can link to episode pages properly